src/main/java/roomescape/theme/ThemeRepository.java

```diff
@@ -0,0 +1,20 @@
+package roomescape.theme;
+
+import java.time.LocalDate;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface ThemeRepository extends JpaRepository<Theme, Long> {
+    @Query(nativeQuery = true,
+            value = """
+                            SELECT t.id, t.name, t.description, t.thumbnail
+                            FROM theme t INNER JOIN reservation r
+                                ON t.id=r.theme_id
+                            WHERE r.date BETWEEN ?1 AND ?2
+                            GROUP BY t.id, t.name, t.description, t.thumbnail
+                            ORDER BY COUNT(*) DESC
+                            LIMIT ?3
+                    """)
```

Comment:
```suggestion
    @Query(nativeQuery = true,
            value = """
                            SELECT t.id, t.name, t.description, t.thumbnail
                            FROM theme t INNER JOIN reservation r
                                ON t.id=r.theme_id
                            WHERE r.date BETWEEN :from AND :to
                            GROUP BY t.id, t.name, t.description, t.thumbnail
                            ORDER BY COUNT(*) DESC
                            LIMIT :size
                    """)
```
가독성을 위해 쿼리 내부에서 파라미터 값을 이름 기반으로 받아오는건 어떨까요?

Reply:
이름 기반 바인딩으로 수정하겠습니다!